src/gui/raster/qgssinglebandpseudocolorrendererwidget.cpp

```diff
@@ -576,10 +710,32 @@ void QgsSingleBandPseudoColorRendererWidget::on_mColormapTreeWidget_itemDoubleCl
   }
   else
   {
+    if ( column == 2 )
```

Comment:
It'd be a good idea to move these hardcoded column numbers to a private enum and use the enum values instead of the raw integers, just in case the columns are shifted around in future